api-testing/multicloud-api/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java'
+}
+
+group 'www.opensds.io'
+version '1.0.0'
```

Comment:
what is this version means?

Reply:
Its simple build version [Link](https://en.wikipedia.org/wiki/Software_versioning)

Reply:
Is it the version from gradle...? Just want to know whether that needs to be maintained or not for every changes or not

Reply:
How are we deciding for versions on this artifacts and module?

Reply:
@skdwriting Not for every changes. If we follow maintain build version or we need to track in future then. Other wise no need to change.

Reply:
> How are we deciding for versions on this artifacts and module?

Version already given.

Reply:
0.1.0 if its initial version. Thanks

Reply:
I have changed. Please review.